web_app/db/crud/user.py

```diff
@@ -215,3 +216,53 @@ def delete_user_by_wallet_id(self, wallet_id: str) -> None:
                 session.rollback()
                 logger.error(f"Failed to delete user with wallet_id {wallet_id}: {e}")
                 raise e
+
+    async def withdraw_all(self, wallet_id: str) -> Dict[str, any]:
+        """
+        Withdraws all available tokens (ETH, STRK, USDC) for a given wallet.
+        
+        Args:
+            wallet_id: The ID of the wallet to withdraw from
+            
+        Returns:
+            Dict containing withdrawal results for each token
+            
+        Raises:
+            HTTPException:
+                404: If user not found
+                400: If contract is not deployed
+                500: For database or withdrawal errors
+        """
+        try:
+            user = self.get_user_by_wallet_id(wallet_id)
+            if not user:
```

Comment:
DB is not the right place to raise any  server exception, that is why I left comment in endpoint not here